techeria_controller/techeria_app/views.py

```diff
@@ -1,7 +1,7 @@
 from django.db.models.fields import NullBooleanField
 from django.http import response
 from django.shortcuts import redirect, render
-from techeria_app.models import Products, BuyerModel, SellerModel
+from techeria_app.models import BuyerModel, SellerModel, Products
```

Comment:
I am having using issues with the seller and buyer model tables. I think we have to pay attention about the description, meta data before linking them